tools/topology/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ set(TPLGS
 	"sof-hda-generic\;sof-hda-generic-4ch\;-DCHANNELS=4\;-DPPROC=volume\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4"
 	"sof-hda-generic\;sof-hda-generic-eq-2ch\;-DCHANNELS=2\;-DPPROC=eq-iir-eq-fir-volume\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4"
 	"sof-hda-generic\;sof-hda-generic-eq-4ch\;-DCHANNELS=4\;-DPPROC=eq-iir-eq-fir-volume\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4"
+	"sof-hda-generic\;sof-hda-generic-tdfb_50mm-2ch\;-DCHANNELS=2\;-DPPROC=volume\;-DDMIC16KPROC=tdfb-volume\;-DDMIC16KPROC_FILTER1=tdfb_coef_line2_50mm_pm25deg_16khz.m4\;-DDMICPROC=tdfb-volume\;-DDMICPROC_FILTER1=tdfb_coef_line2_50mm_pm25deg_48khz.m4"
```

Comment:
@juimonen Tested that this topology is more silent than a plain sof-hda-generic. I noticed in topology for UP2 a -6 dB level change vs. normal topology. I'll check the filters that there's no extra 6 dB attenuation in place.

Reply:
Done, there was a FIR shift value polarity issue in hifi3 version.